amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/job/DispositionLifecycleJobExecuter.java

```diff
@@ -61,7 +61,7 @@
 public class DispositionLifecycleJobExecuter extends RecordsManagementJobExecuter
 {
     /** logger */
-    private static final Log logger = LogFactory.getLog(DispositionLifecycleJobExecuter.class);
+    private static Log logger = LogFactory.getLog(DispositionLifecycleJobExecuter.class);
```

Comment:
This is an interesting one - I always mark loggers as final, but I don't have a good reason for it.  Let's revisit the logger in a separate PR anyway!